thrift-bindings/remote_controller.thrift

```diff
@@ -49,6 +66,60 @@ service RemoteController {
     Cluster splitMemberFromCluster(1:string memberId)
     Cluster mergeMemberToCluster(1:string clusterId, 2:string memberId)
 
+    /**
+     * Reads the environment variables and calls loginToHazelcastCloud() method with these variables.
+     * @throws CloudException
+     */
+    void loginToHazelcastCloudUsingEnvironment() throws (1:CloudException cloudException)
+
+    /**
+     * Logins the hazelcast cloud, sets the bearer token, uri and baseUrl to HazelcastCloudManager then it will be ready to use cloud API
+     * @throws CloudException
+     */
+    void loginToHazelcastCloud(1:string uri, 2:string apiKey, 3:string apiSecret) throws (1:CloudException cloudException)
+
+    /**
+     * Creates a standard cluster
+     * @return CloudCluster
+     * @throws CloudException
+     */
+    CloudCluster createHazelcastCloudStandardCluster(1:string hazelcastVersion, 2:bool isTlsEnabled) throws (1:CloudException cloudException)
+
+    /**
+     * Scaling up and down a given cluster
```

Comment:
Please also document the parameters for the methods. E.g. what is `id`? In executeOnController there is a parameter called clusterId, are they different?

Reply:
When it is generated by thrift, it is written in the generated file. When we also specify params in thrift file, it will be written twice in the genereted file `RemoteControler.java`

`executeOnController` is not relevant with cloud, it is a method which is already there